api/src/pcapi/validation/models/offer.py

```diff
@@ -5,9 +5,10 @@
 
 def validate(offer: Offer, api_errors: ApiErrors) -> ApiErrors:
     venue = offer.venue if offer.venue else find_venue_by_id(offer.venueId)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question (curiosité):** ‏ça existe des offres sans venue ?
`offer.venue_id` peut ne pointer nulle part ? Pas de on_delete cascade ?

Reply:
J'ai moi aussi tiqué sur cette ligne. :) Je pense (???) qu'on gère ici le cas où on a un objet `Offer` non enregistré en base de données, sur lequel on a enregistré `venueId` mais que `venue` n'est pas accessible. Exemple dans un `flask shell` : 

```python
>>> venue = Venue.query.first()
>>> offer = Offer(venueId=venue.id)
>>> offer.venueId
977
>>> offer.venue is None
True
```

C'est pas formidable, comme bout de code...